etcdctl/README.md

```diff
@@ -344,13 +344,16 @@ LEASE TIMETOLIVE retrieves the lease information with the given lease ID.
 # lease 2d8257079fa1bc0c granted with TTL(500s)
 
 ./etcdctl put foo1 bar --lease=2d8257079fa1bc0c
+#OK
+
 ./etcdctl put foo2 bar --lease=2d8257079fa1bc0c
+#OK
 
 ./etcdctl lease timetolive 2d8257079fa1bc0c
 # lease 2d8257079fa1bc0c granted with TTL(500s), remaining(481s)
 
 ./etcdctl lease timetolive 2d8257079fa1bc0c --keys
-# lease 2d8257079fa1bc0c granted with TTL(500s), remaining(472s), attached keys([foo2 foo1])
```

Comment:
@sinsharat Yes please. We don't sort keys in response unless specified.

Thanks!

If you want to look at the source code

https://github.com/coreos/etcd/blob/master/lease/lessor.go#L483-L490